recipes/veloxchem/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "veloxchem" %}
+{% set version = "1.0" %}
+{% set pre = "rc3" %}
+{% set sha256 = "ed22aeb72d832f0daccffa393a1a349dab65fc85cefbb6a069933d3b51ad7ffd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/veloxchem/veloxchem/-/archive/v{{ version }}-{{ pre }}/veloxchem-v{{ version }}-{{ pre }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/geometric-as-posix.patch
+
+build:
+  number: 0
+  script_env:
+    - OMPI_MCA_plm_rsh_agent=  # [mpi == 'openmpi' and linux]
+  binary_relocation: true
+  entry_points:
+    - vlx=veloxchem.main:main
+  run_exports:
+    - {{ pin_subpackage('veloxchem', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake >=3.18
+    - {{ compiler('cxx') }}
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx or win]
+    - ninja
+    - pkg-config  # [unix]
+    - pybind11 >=2.6,<3.0
+  host:
+    - python
+    - pybind11-abi
+    - {{ mpi }}
+    - mpi4py >=3.0
+    - libcblas
+    - liblapacke
+    - numpy >=1.15
+    - xtb  # [unix]
+  run:
+    - psutil >=5.6
+    - mdanalysis >=1.0
+    - geometric
+    - cppe  # [unix]
+    - h5py >=2.8
+    - python
+    - {{ pin_compatible('mpi4py') }}
+    - {{ pin_compatible('numpy') }}
+  run_constrained:
+    - libopenblas * openmp*  # [unix]
+
+test:
+  requires:
+    - pytest
+    - libopenblas  # [unix]   
+    - mkl  # [win]
+  imports:
+    - veloxchem
+  commands:
+    # verify existence
+    - test -d $SP_DIR/veloxchem  # [unix]
+    - ls -l $SP_DIR/veloxchem  # [unix]
+    - test -f $PREFIX/bin/vlx  # [unix]
+    - if not exist %SP_DIR%\\veloxchem exit 1  # [win]
+    - dir %SP_DIR%\\veloxchem  # [win]
+    - if not exist %SCRIPTS%\\vlx.exe exit 1  # [win]
```

```suggestion
    - pip check
```
These are all redundant with the python `imports` test above and the calls to `vlx --version` below. However, we do want pip to check that your python dependencies are all consistent.